includes/wikia/models/NavigationModel.class.php

```diff
@@ -82,6 +82,15 @@ public function clearMemc( $key = self::WIKIA_GLOBAL_VARIABLE ) {
 		WikiaDataAccess::cachePurge(
 			$this->getMemcKey( $key )
 		);
+
+		// Purging mcache set by invocation of NavigationModel::getTree() by NavigationModel::getLocalNavigationTree();
+		WikiaDataAccess::cachePurge(
+			$this->getTreeMemcKey( self::TYPE_MESSAGE, self::WIKI_LOCAL_MESSAGE, implode( [
+				self::LOCALNAV_LEVEL_1_ITEMS_COUNT,
+				self::LOCALNAV_LEVEL_2_ITEMS_COUNT,
+				self::LOCALNAV_LEVEL_3_ITEMS_COUNT
+			], '-' ) )
```

Comment:
yes, I missed that method. And according to the doc you linked `implode() can, for historical reasons, accept its parameters in either order. For consistency with explode(), however, it may be less confusing to use the documented order of arguments.`